src/pages/my_page/user_profile/UserIdentity.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+import { Box, IconButton, TextField, Tooltip, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { BackgroundDiv } from '../../../components/styled/Wrappers';
+import { User } from '../../../types/user.types';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import { getMaskedNationalIdentityNumber } from '../../../utils/user-helpers';
+import { UserRoles } from './UserRoles';
+
+interface UserIdentitiesProps {
+  user: User;
+}
+
+export const UserIdentity = ({ user }: UserIdentitiesProps) => {
+  const { t } = useTranslation();
+  const fullName = `${user.givenName} ${user.familyName}`;
+  const userCristinId = user.cristinId?.split('/').at(-1);
```

Comment:
Nice å bruke nye funksjoner som `at()`! Bare et lite tips når man skal bruke nye ting er å sjekke caniuse.com for å se hvor mange nettlesere som takler det, og hor mange som bruker den.

I dette tilfellet er den støttes av 88% av brukere, som jeg syns er akkurat innafor: https://caniuse.com/mdn-javascript_builtins_array_at

Reply:
alternativet ville vært å feks bruke `.pop()`, som støttes for 96% https://caniuse.com/mdn-javascript_builtins_array_pop